packages/dscc-gen/src/config.ts

```diff
@@ -249,7 +249,7 @@ const withMissing = async (
         connectorDefaults
       );
     case ProjectChoice.VIZ:
-      await checkGsutilInstalled();
+      // await checkGsutilInstalled();
```

Comment:
remove instead of comment

Reply:
same as above- i think we should keep this; it was commented out for debugging.